code/modules/projectiles/guns/projectile/automatic/lmg.dm

```diff
@@ -26,6 +26,8 @@
 	one_hand_penalty = 30 //you're not Stallone. LMG level.
 	spawn_blacklisted = TRUE
 	rarity_value = 80
+	slowdown_hold = 4
```

Comment:
Tune down the maxim to 2 as well. The brace penalty is more than enough punishment as is.
```suggestion
	slowdown_hold = 0.5
```